src/main/java/net/sf/jabref/preferences/PreviewPreferences.java

```diff
@@ -0,0 +1,108 @@
+package net.sf.jabref.preferences;
+
+import java.util.List;
+import java.util.Map;
+
+import net.sf.jabref.logic.citationstyle.CitationStyle;
+
+import static net.sf.jabref.Globals.prefs;
+
+
+public class PreviewPreferences {
+
+    private static final String CYCLE_PREVIEW_POS = "cyclePreviewPos";
+    private static final String CYCLE_PREVIEW = "cyclePreview";
+    private static final String PREVIEW_PANEL_HEIGHT = "previewPanelHeight";
+    private static final String PREVIEW_STYLE = "previewStyle";
+    private static final String PREVIEW_ENABLED = "previewEnabled";
+
+    private final JabRefPreferences preferences;
+
+
+    public PreviewPreferences(JabRefPreferences preferences) {
```

Comment:
Mhhh... I would suggest to always read/write all the preference values. With this you have a performance overhead but the code looks better.